build/mediaelement-and-player.js

```diff
@@ -1789,6 +1789,7 @@ mejs.YouTubeApi = {
 
 					mejs.YouTubeApi.createEvent(player, pluginMediaElement, 'loadstart');
 					mejs.YouTubeApi.createEvent(player, pluginMediaElement, 'canplay');
+				  mejs.YouTubeApi.createEvent(player, pluginMediaElement, 'loadedmetadata');
```

Comment:
i don't know why this indentation doesn't match that of the source.  maybe your source originally had this indentation, then you built it, then fixed the source and committed without remembering to re-build it?